src/components/NodeList/NodeListItem.tsx

```diff
@@ -111,56 +114,102 @@ const Address = styled(PlaceName)`
 `
 
 type Props = {
+  children?: React.ReactNode
+  highlightPointOnHover?: boolean
+  isActive?: boolean
   nodeId: string
-  title: string
-  address: string,
-  placeName: string
+  nodeLocationId: string
   onClick?: (id: string) => void
-  isActive?: boolean
-  children?: React.ReactNode
+  placeName: ReactNode
+  title: string
 }
 
-const NodeListItem = ({
+export const NodeListItem = ({
+  children,
+  highlightPointOnHover = false,
+  isActive,
   nodeId,
-  title,
-  address,
+  nodeLocationId,
+  onClick,
   placeName,
-  onClick: onClickProp,
-  isActive,
-  children,
+  title,
   ...props
 }: Props) => {
-  const onClick = useCallback(() => {
-    if (typeof onClickProp === 'function') {
-      onClickProp(nodeId)
-    }
-  }, [onClickProp, nodeId])
-
   const transition = useTransition(isActive, {
     from: { opacity: 0 },
     enter: { opacity: 1 },
     leave: { opacity: 0 },
   })
 
+  const ref = useRef<HTMLDivElement>(null)
+
+  useEffect(() => {
+    let mounted = true
+
+    if (isActive) {
+      setTimeout(function scrollReffedElementIntoView() {
```

Comment:
Each node list element takes care of its own "auto-scroll into view" behaviour. I dropped the `data-node-id` custom attr. Too much magic.